src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Figure implements Drawable, AreaCalculator {
```

Comment:
`Figure` must be a abstract
```suggestion
public abstract class Figure implements Drawable, AreaCalculator {
```